assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -240,25 +241,50 @@ const SelectionCardContainer = ({
 
 const RoutePatternLayers = ({
   routePattern,
+  isSelected,
   onShapeClick,
 }: {
   routePattern: RoutePattern
+  isSelected: boolean
   onShapeClick: () => void
 }): JSX.Element => {
   return routePattern.shape ? (
     <>
-      <RouteShape shape={routePattern.shape} onClick={onShapeClick} />
       <ZoomLevelWrapper>
         {(zoomLevel) => {
+          const routePatternIdSplit = routePattern.id.split("-")
+          const variantFormatted =
+            routePatternIdSplit.length === 3 && routePatternIdSplit[1] !== "_"
+              ? "_" + routePatternIdSplit[1]
+              : ""
           return (
             <>
               {routePattern.shape && (
-                <RouteStopMarkers
-                  stops={routePattern.shape.stops || []}
-                  includeStopCard={true}
-                  direction={routePattern.directionId}
-                  zoomLevel={zoomLevel}
-                />
+                <>
+                  <RouteShape
+                    shape={routePattern.shape}
+                    isSelected={isSelected}
+                    onClick={onShapeClick}
+                  >
+                    {zoomLevel >= 16 && !isSelected ? (
+                      <Tooltip
+                        className="route-shape__tooltip"
+                        sticky={true}
+                        direction="top"
+                      >
+                        <>
+                          {`Click to select route ${routePattern.routeId}${variantFormatted}.`}
```

Comment:
**suggestion:** I think you might be able to do this without string interpolation or even a fragment